spec/support/csv_file_helper.rb

```diff
@@ -43,6 +43,20 @@ module CSVFileHelper
     link_type: 'LinkType',
   }.freeze
 
+  TRUST_ATTRS = {
```

Comment:
This might be over-engineering, but if this constant lived on `TrustDataFile`, then you could use it both in the model and the specs...

Reply:
Nice idea